docs/package_templates/meson_package/all/conanfile.py

```diff
@@ -110,6 +112,7 @@ def generate(self):
         tc = MesonToolchain(self)
         # In case need to pass definitions directly to the compiler
         tc.preprocessor_definitions["MYDEFINE"] = "MYDEF_VALUE"
+        tc.project_options["feature"] = "enabled" if self.options.get_safe("feature") else "disabled"
```

Comment:
It could use a lambda in case having more options, but it's totally fine right now.

```
enabled_disabled = lambda opt: "enabled" if opt else "disabled"
```